Updates to rich text editor Umbraco 8 (#2275)
umbraco/UmbracoDocs: Tutorials/Editors-Manual/Working-with-Content/Rich-Text-Editor/index.md

5. In the Title field, enter a name/description for the image (by default the name of the file will be automatically entered for your convenience). It is important to add descriptive titles to images as these as used to assist visually impaired users.

> 6. Click ***Insert*** to add the image.

Reviewer: ```suggestion
6. Click ***Select*** to add the image.
```

5. In the Title field, enter a name/description for the image. (By default the name of the file will be automatically entered for your convenience).

> 6. Click ***Insert** to add the image.

Reviewer: ```suggestion
6. Click ***Select*** to add the image.
```

> ![mediaUpload.jpg](images/Upload-media.png)

Reviewer: This image - I'm not sure where it belongs?
It's an image of the search feature. Perhaps it should have been another image here? 🤔

2. Click the ***Insert Macro*** button from the toolbar to open the macro properties dialog box.
3. Select a macro to insert from the Choose a macro drop-down list.
4. Configure the macro as required.
5. Click ***Submit*** to insert the macro.
6. The macro is displayed in the editor with a yellow dotted border.

> ![macroPicker.jpg](images/Macro-picker-button.png)

Reviewer: In this image, the "Insert Form" macro has already been selected - is that intentional?
As I read the steps, this image is supposed to show me how to pick a macro, right? Or perhaps I'm missing something!

![macroPicker.jpg](images/Macro-picker-button.png)

![macroPicker.jpg](images/Select-a-macro.png)

> *note that if there is no macros to chose from it will not look like this- this is from the starter kit package*

Reviewer: Thanks for getting the image updated 😄 

About the note here .. Shouldn't we use the note formatting instead?
```suggestion
:::note
If there is no macros to choose from it will not look like this. What you see above, is how it looks when the Starter Kit package is installed.
:::
```
